ios/ProjectAccessibility/Info.plist

```diff
@@ -44,9 +44,11 @@
 	<key>NSPhotoLibraryAddUsageDescription</key>
 	<string/>
 	<key>NSLocationAlwaysUsageDescription</key>
-  <string>Need to track user for geofencing</string>
+  	<string>Need to track user for geofencing</string>
 	<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
-  <string>Need to track user for geofencing</string>
+  	<string>Need to track user for geofencing</string>
+	<key>NSMicrophoneUsageDescription</key>
+	<string>Give $(PRODUCT_NAME) permission to use your microphone. Your record wont be shared without your permission.</string>
```

Comment:
wont -> won't

Reply:
Waarom is het ook in het Engels?

Reply:
ik merkte bij location description dat het in het engels was, dus had ik dat overgenomen, maar sinds ik aan kan nemen dat ze niet in het buitenland iets willen gaan doen met de app. Zet ik het in het nederlands, zelfde voor locatie descriptie